burst-gradle-plugin/src/test/projects/multiplatform/build.gradle.kts

```diff
@@ -20,4 +23,15 @@ allprojects {
     mavenCentral()
     google()
   }
+
+  tasks.withType(JavaCompile::class.java).configureEach {
+    sourceCompatibility = "1.8"
```

Comment:
None of these projects built on my machine without this. My default JDK is 23, but Kotlin doesn't know about that version so it drops down to target 22 and then complains Kotlin and Java targets don't match.